Style.css

```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+header div {
+  width: 375px;
+  height: 40px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  font-size: large;
+  margin: auto;
+  width: 375px;
+}
+
+.logo {
+  font-family: inter, sans-serif;
+  font-weight: bold;
+  flex-grow: 0;
```

Comment:
- [ ] As suggested by all the previous reviewers, I notice what they requested you work on is still undone by you. Currently, your project is not aligning very well with the mobile screen size, and the goal of this project is to work on the project in mobile screen size and the ability to replicate the design given to you by a client, with this, any client can be more confident in you to deliver a better job for him.

- [ ] My humble advice is to please work on your project alignment and make sure it fits perfectly to the mobile screen size.

Your design 👇 

![only](https://user-images.githubusercontent.com/78634841/212262304-76624b85-ed8d-488c-98d4-8a5ebcbb50f4.png)

Figma design 👇 

![temp real](https://user-images.githubusercontent.com/78634841/212262666-cd9230c2-614a-448d-a0cb-ae4637d675a4.png)